open_ce/build_tree.py

```diff
@@ -49,13 +48,13 @@ def __str__(self):
         return "({} : {})".format(self.packages, self.build_command)
 
     def __hash__(self):
-        return self._hash_val
+        return hash(self.build_command)
```

Comment:
I had previously moved this from calculating the hash each time `__hash__` was called to a saved value within `self._hash` because I was seeing this hash function taking up some amount of time in the performance printouts.

However, it does assume that the build_command is never modified. I think that is a reasonable assumption, but if not, you can leave your change in.

Reply:
Okay, I think it makes sense to hash that hash value. I just saw that it was using the name and reduced it down. I'll put it back.

Reply:
The hash does get called ALOT, because of the graph composition.